src/coreclr/tools/Common/Compiler/DependencyAnalysis/Target_X64/X64Emitter.cs

```diff
@@ -174,6 +174,15 @@ public void EmitRETIfEqual()
             Builder.EmitByte(0xC3);
         }
 
+        public void EmitXOR(Register regDst, Register regSrc)
```

Comment:
If we want to call this EmitXOR, it should be something along the lines of:

```csharp
public void EmitXOR(Register regDst, ref AddrMode memory)
{
    EmitIndirInstructionSize(0x32, regDst, ref memory);
}
````

Someone might use this to `builder.EmitXOR(Register.RAX, Register.RCX)` and this actually emitting `xor eax, ecx` (32 bit regs) is subtle.

Or just rename this to `EmitZeroReg` and drop the second parameter?

Reply:
I renamed to `EmitZeroReg` since that's what we really need in this PR. Thanks!